src/charm.py

```diff
@@ -300,6 +303,20 @@ def _start_opensearch(self) -> bool:
             self.unit.status = BlockedStatus(" - ".join(missing_sys_reqs))
             return False
 
+        # check if there are shards that are "busy" or "relocating"
+        # defer the start until all the shards are "started"
```

Comment:
```suggestion
# OpenSearch automatically finds and adds shards. Meaning  that when a unit starts OpenSearch will automatically add it to the cluster. To prevent overloading the cluster, shards must be added one at a time. To insure this it is necessary to defer starting a shared until all shards have been "started" (i.e. they are not in the "busy" or "relocating" state)
```
^ add this comment and then wrap to 99 char length